### PR TITLE
Grammar Mistake 😂

### PR DESCRIPTION
**Pull request by Ali-Raza-Arain**

### CODE REVIEW

**Comment by Ali-Raza-Arain** [comment]
?

**Comment by Ali-Raza-Arain** [comment]
> ## Summary
> ## How did you test this change?

Simple, Using my skills...

File: README.md [hunk 1]
```diff
@@ -52,23 +52,23 @@ root.render(<HelloMessage name="Taylor" />);
...
 
 ### [Contributing Guide](https://reactjs.org/docs/how-to-contribute.html)
 
-Read our [contributing guide](https://reactjs.org/docs/how-to-contribute.html) to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to React.
+Read our [contributing guide](https://reactjs.org/docs/how-to-contribute.html) to learn about our development process, how to propose bug fixes and improvements, and how to build and test your changes to React.
 
 ### Good First Issues
 
-To help you get your feet wet and get you familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/react/labels/good%20first%20issue) that contain bugs that have a relatively limited scope. This is a great place to get started.
+To help you get your feet wet and get familiar with our contribution process, we have a list of [good first issues](https://github.com/facebook/react/labels/good%20first%20issue) that contain bugs that have a relatively limited scope. This is a great place to get started.
```
Comment (rbalicki2) [inline README.md:71]: I think `and to get you familiar` is the most correct way to phrase this. Could you rephrase it that way, then it looks good to land?

Comment (Ali-Raza-Arain) [reply README.md:71]: Yes, "you" is the most appropriate for this sentence.
